spec/presenters/listing_presenter_spec.rb

```diff
@@ -48,91 +48,156 @@
     let(:booking4) { FactoryGirl.create(:booking, tx: transaction4, start_time: '2017-11-14 09:00', end_time: '2017-11-14 10:00', per_hour: true) }
 
 
-    it '#working_hours_blocked_days 2017-11-14 is busy full day.
+    describe "blocked dates" do
```

Comment:
Reelt set er det vel `#working_hours_blocked_days` der beskrives her? Det er hvertfald den del af `it`-titlerne, som gentages.